src/NinjaTools.FluentMockServer.API/Extensions/LINQ.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Primitives;
+using NinjaTools.FluentMockServer.API.Models;
+
+namespace NinjaTools.FluentMockServer.API.Extensions
+{
+    public static class LINQ
+    {
+
+
+        public static Dictionary<string, string[]> ToDictionary<T>(this T headers) where T : IHeaderDictionary
+        {
+            return headers.ToDictionary(
+                k => k.Key,
+                v => v.Value.ToArray());
+        }
+
+        [NotNull]
+        public static IReadOnlyList<T> EnsureNotNullNotEmpty<T>([CanBeNull] this IEnumerable<T>? input)
+        {
+            var collection = (input ?? throw new InvalidOperationException(nameof(input))).ToList();
+
+            if (!collection.Any())
+                throw new InvalidOperationException("EnsureNotNullNotEmpty<T>; EMPTY SEQUENCE");
+
+            if (collection.Any(item => item is null))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)